spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -70,10 +70,10 @@ module.exports = function (provider, setup) {
       }
       expect(evt).toEqual('onData');
       expect(msg.data.byteLength).toEqual(10);
-      done();
       socket.close(function () {});
       client.close(function () {});
       receiver.close(function () {});
+      done();
```

Comment:
If the above functions are intended to be async, you really want something like an all promise, then call done, otherwise you may have handling state after your done call which could interact badly with other tests.